### DISCUSSION TITLE
May I ask about the container The issue with NewAppTabs or the length adaptation of the Android interface

### DISCUSSION THREAD

**Discussion by cplasfwst** [fyne-io/fyne#4720] [Q&A]
I only created an Android app, but when I set it up like this, the height of my element occupying the interface exceeds the height of the device.

The layout I am using is like this
`zhuye := jiemian.Zhuyemian(myApp, myWindow)
	douyin := jiemian.Douyinxieyi(myApp, myWindow)
	pinglun := jiemian.Pinglun(myApp, myWindow)
	tabs := container.NewAppTabs(
		container.NewTabItem("main", zhuye),
		container.NewTabItem("douyin", douyin),
		container.NewTabItem("tallk", pinglun),
	)

	//tabs.Append(container.NewTabItemWithIcon("Home", theme.HomeIcon(), widget.NewLabel("Home tab")))

	tabs.SetTabLocation(container.TabLocationLeading)

	myWindow.SetContent(tabs)
	myWindow.ShowAndRun()`

There are many elements in my pinglun page that exceed the height of the device, and the ones below block the display. How can I set the pinglun page to slide down to see more buttons or elements?
![wenti-fyne](https://github.com/fyne-io/fyne/assets/34654697/d671e84c-a9f0-4903-969e-55ef881b6036)
For example, the red color I circled and the buttons. How can I make the interface slide down to continue displaying

**Comment by cplasfwst** [comment]
I seem to have found a parameter that can be used, but after adding it, it becomes possible to slide left and right. I just want to slide up and down

`tabs := container.NewAppTabs(
		container.NewTabItem("main", zhuye),
		container.NewTabItem("douyin", douyin),
		container.NewTabItem("talk", container.NewVScroll(pinglun)),
	)`

container.NewVScroll.It can be used, but it has increased my width
![029E52EE80A11EC1251F4ABF1BE02054](https://github.com/fyne-io/fyne/assets/34654697/20b069d2-0f08-4919-ae98-5226de1092a0)

**Comment by andydotxyz** [comment]
It seems like making it scrollable would be the solution. So wrap your content in `container.NewScroll`
